src/Analysis/GWASApp/Components/MakeFullscreenButton/MakeFullscreenButton.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+import Button from '@gen3/ui-component/dist/components/Button';
+import FullscreenClassNames from './FullscreenClassNames';
+import './MakeFullscreenButton.css';
+
+const MakeFullscreenButton = () => {
+  const [analysisIsFullscreen, setAnalysisIsFullscreen] = useState(false);
+
+  const setElementsDisplay = (selector, displayValue) => {
+    document.querySelectorAll(selector).forEach((element) => {
+      const temporaryElement = element;
```

Comment:
Minor comment: is this assignment statement necessary? Could just change the styling on 'element'